source/developer/interactive-dialogs.rst

```diff
@@ -210,6 +210,36 @@ The list of supported fields for the ``select`` type element is included below:
     "default", "String", "(Optional) Set a default value for this form element. Maximum 3,000 characters."
     "placeholder", "String", "(Optional) A string displayed to help guide users in completing the element. Maximum 3,000 characters."
 
+Checkbox Element
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Checkbox element are single-line plain text fields with a Checkbox to be selected. Below is an example of a Checkbox element that asks for meeting feedback.
+
+.. image:: ../../source/images/interactive-dialog-bool.png
+
+.. code-block:: json
+
+  {
+    "display_name": "Can you please select below",
+    "placeholder": "The meeting was helpful.",
+    "name": "meeting_input",
+    "type": "bool",
+  }
+
+The full list of supported fields are included below:
+
+.. csv-table::
+    :header: "Field", "Type", "Description"
+
+    "display_name", "String", "Display name of the field shown to the user in the dialog. Maximum 24 characters."
+    "name", "String", "Name of the field element used by the integration. Maximum 300 characters. You should use unique “name” fields in the same dialog."
+    "type", "String", "Set this value to ``bool`` for a checkbox element."
+    "optional", "Boolean", "(Optional) Set to true if this form element is not required. Default is ``false``."
+    "help_text", "String", "(Optional) Set help text for this form element. Maximum 150 characters."
+    "default", "String", "(Optional) Set a default value for this form element. True or false"
+    "placeholder", "String", "(Optional) A string displayed to include a label besides the checkbox. Maximum 150 characters."
```

Comment:
What is the difference between `placeholder` and `help_text`?

Reply:
I believe `help_test` appears below the field while `placeholder` appears to the right of the checkbox.

as per https://github.com/mattermost/mattermost-webapp/blob/master/components/widgets/settings/setting.jsx#L29

and https://github.com/mattermost/mattermost-webapp/pull/3066/files#diff-c3112a2a9bea3269d2250feb71051a89R47

Reply:
I leaved a comment about this in the original PR:
https://github.com/mattermost/mattermost-webapp/pull/3066#discussion_r303875873